src/runtime/runtime.js

```diff
@@ -1160,6 +1168,17 @@ export class ConfiguredRuntime {
   showBestAudienceAction() {
     warn('Not implemented yet');
   }
+
+  /** @override */
+  showMeter() {
+    return this.documentParsed_.then(() => {
+      this.setOnNativeSubscribeRequest(() => {
+        this.showOffers({isClosable: true});
+      });
+      const flow = new MeterFlow(this);
+      return flow.start();
```

Comment:
Could you add a test for this?
https://app.codecov.io/gh/subscriptions-project/swg-js/compare/2050/diff

Reply:
Reverted.